### PR TITLE
fix: item_template bonding to account explained to dummies

### PR DESCRIPTION
**Pull request by BarbzYHOOL**
Just read what I proposed, it's interesting!

Finally this system is now explained to everybody xD

### CODE REVIEW

File: docs/item_template.md [hunk 1]
```diff
@@ -2336,7 +2336,7 @@ Bitmask field that contains flags that the item has on it. As all other such fie
 <tr class="even">
 <td><p>134217728</p></td>
 <td>0x08000000</td>
-<td><p>Bind to Account (Set Quality = 7 for the corresponding color. Might require to set Bonding = 1)</p></td>
+<td><p>Bind to Account (Requires to set Bonding > 0)</p></td>
```
Comment (FrancescoBorzi) [inline docs/item_template.md:2339]: why removing the tip about the color?

Comment (BarbzYHOOL) [reply docs/item_template.md:2339]: see my previous PR (it should have been in previous PR)

Comment (BarbzYHOOL) [reply docs/item_template.md:2339]: see here https://github.com/azerothcore/wiki/pull/220

Comment (Kitzunu) [reply docs/item_template.md:2339]: Are you sure that you can still have Bind to Account without Q = 7? Never actually tested it myself

Comment (BarbzYHOOL) [reply docs/item_template.md:2339]: yes and many items are like that by default (example: murloc costume)

**Comment by BarbzYHOOL** [comment]
i approve my own PR

File: docs/item_template.md (hunk 1 above)
Comment (Kitzunu) [reply docs/item_template.md:2339]: Ah yes that is true, now I remember

**Comment by Helias** [comment]
Working on Keira3 I inspected the related values of this field, and it seems ok the description for me.

**Pull request merged**